neuralop/layers/attention_kernel_integral.py

```diff
@@ -0,0 +1,194 @@
+import torch
+from torch import nn
+import math
+from torch.nn.init import xavier_uniform_, zeros_
+
+
+class AttentionKernelIntegral(torch.nn.Module):
+    """
+    Kernel integral transform with attention
+    Computes \int_{Omega} k(x, y) * f(y) dy,
+    where:
+          K(x, y) = \sum_{c=1}^d \q_c(x) * \k_c(y)
+          f(y) = v(y)
+
+    Parameters
+    ----------
+    channels : int, input and output channels
```

Comment:
Do we not want to have allow them to be different? 
Nitpick: for consistency with other layers would be great to call then either in_channels and out_channels or just `n_channels`.

Reply:
Yes definitely and that's more reasonable. I often omit it since input and output channels in attention are almost always the same to accommodate the residual